pkg/install/condition.go

```diff
@@ -62,6 +64,11 @@ func (i *Installer) clusterVersionReady() (bool, error) {
 func (i *Installer) ingressControllerReady() (bool, error) {
 	ic, err := i.operatorcli.OperatorV1().IngressControllers("openshift-ingress-operator").Get("default", metav1.GetOptions{})
 	if err == nil && ic.Status.ObservedGeneration == ic.Generation {
+		if _, ok := i.env.(env.Dev); !ok {
+			if ic.Spec.DefaultCertificate == nil {
```

Comment:
I don't understand how this makes any difference.  Right before calling ingressControllerReady, we call configureIngressCertificate, which sets ic.Spec.DefaultCertificate non-nil.  In what circumstances do we get here with ic.Spec.DefaultCertificate nil?